javascript/modules/client/src/context.ts

```diff
@@ -140,6 +144,17 @@ export class Context {
     }
   }
 
+  useIpfsNodes(nodes: Options[]): IPFSHTTPClient[] {
+    if (nodes.length > 0) {
```

Comment:
Should we accept a 0 length array? I'd throw an error

Reply:
I think so, the library have default values if not specified pointing to localhost, so we can keep this behavior, or force the user to always send a configuration, so this array will never be 0